### PR TITLE
Deployment fixups

### PR DESCRIPTION
**Pull request by spookyrecharge**

### CODE REVIEW

File: roles/services/crawler/templates/config.yml [hunk 1]
```diff
@@ -9,11 +9,16 @@ opensearch:
...
+  addresses:
+  # struggled with that part
+  # at first there was an error regarding 'redis' group does not exist
+  # but later 'ansible.vars.hostvars.HostVarsVars object' has no attribute 'ansible_all_ipv4_addresses'
+    - '172.16.0.2:7000' # redis-1
+    - '172.16.0.2:7001' # redis-1
+    - '172.16.0.3:7000' # redis-2
+    - '172.16.0.3:7001' # redis-2
+    - '172.16.0.4:7000' # redis-3
+    - '172.16.0.4:7001' # redis-3
```
Comment (spookyrecharge) [inline roles/services/crawler/templates/config.yml:21]: it is a temp solution, I just wanted to make sure that the whole cycle is working. Let's discuss it as soon as you're available

File: roles/vendor/opensearch/tasks/opensearch.yml [hunk 2]
```diff
@@ -57,9 +57,10 @@
       group: "{{ os_user }}"
       mode: "u=rwX,g=rwX"
       remote_src: true
+    notify:
```
Comment (spookyrecharge) [inline roles/vendor/opensearch/tasks/opensearch.yml:60]: it is impossible to use 'notify' in a 'block' segment. I think it should be there?

File: roles/setup/hetzner_install_image/tasks/main.yml [hunk 3]
```diff
@@ -11,7 +11,7 @@
     reboot_timeout: 7200
```
Comment (spookyrecharge) [inline roles/setup/hetzner_install_image/tasks/main.yml:11]: overkill number?

Comment (dokterbob) [reply roles/setup/hetzner_install_image/tasks/main.yml:11]: I agree. It seemed that reboot was never working (at least for me). This high number was the result of trying.

File: .gitignore [hunk 4]
```diff
@@ -17,3 +17,8 @@ Thumbs.db
 
 # Text editor stuff
 *.swp
+
+# Dependencies installed by galaxy
+roles/wireguard
+roles/golang
+roles/elasticsearch
```
Comment (dokterbob) [inline .gitignore:24]: Are they installed in `roles` for you?

For me, galaxy installs to `~/.ansible/roles`. Do you think you might find a way to accomplish the same for you? That way, we won't have to (and might potentially forget to) update gitignore every time deps change.

Comment (dokterbob) [reply .gitignore:24]: Note that Elasticsearch can now actually be removed from deps!

File: roles/services/crawler/templates/config.yml (hunk 1 above)
Comment (dokterbob) [reply roles/services/crawler/templates/config.yml:21]: In the playbook, `gather_facts` needs to be called on the redis hosts. In addition, as some hosts failing should not cause the crawler deployment to fail in addition, we should check in the loop whether this variable is set.

Please see https://github.com/ipfs-search/ipfs-search-deployment/pull/34/files#diff-c8fd42263abb143537dc06f2d96d6ee550458e05a97c3b208f3357a38e6bb896R2 as an example of this.

File: roles/setup/hetzner_install_image/tasks/main.yml (hunk 3 above)
Comment (dokterbob) [reply roles/setup/hetzner_install_image/tasks/main.yml:11]: Did the reboot work in the end? Feel free to suggest a different value or to resort to the default if you think this makes sense.

**Review by dokterbob — CHANGES_REQUESTED** [review]
Just change the redis loop, test it and I think we're good to go.

Sorry for the delay in the feedback.

**Pull request merged**